dockerfiles/compilation/java-8/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV GIT_VERSION 1:2.1.4-2.1
 
 # Install Maven
 WORKDIR /root
-RUN wget http://mirrors.ircam.fr/pub/apache/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.tar.gz
+RUN wget https://archive.apache.org/dist/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.tar.gz
```

Comment:
there's a trick for that, see what yann did for tika here : https://ci.linagora.com/linagora/lgs/openpaas/linagora.esn.tika/blob/master/docker-tikaserver/Dockerfile

Reply:
eurk. Do you think it's really needed? Ah you approved so I guess no :)